FrontEnd/src/views/AddNgoView.vue

```diff
@@ -142,10 +161,60 @@ let newNgo = reactive({
   location: "",
   pincode: "",
 });
+
 const store = useNgoStore();
 
+let msg = reactive([]);
+
 const postNgo = async () => {
-  await store.postNgo(newNgo);
-  return router.push("/ngos");
+  let validationRegex = {
+    name: /^[a-zA-Z]+(\s[a-zA-Z]+)?$/,
+    pincode: /^[0-9]{6}$/,
+    number: /^([+]\d{2})?\d{10}$/,
+    email: /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/,
+  };
+
+  if (newNgo.pincode.match(validationRegex.pincode)) {
```

Comment:
we can have all the regex like above in the helper file as it is now. but we can create separate tiny functions called, 
validatePincode where we will pass the property from the component and here in helper.js function it will just validate it and return back the message using this input param as it is now.

We can do samething for phoneno and email too.
Then we can have a common validateNames(Where we can validate all the rest of the fields).